apkid/rules/dex/obfuscators.yara

```diff
@@ -109,3 +109,34 @@ rule bitwise_antiskid : obfuscator
     is_dex and
     any of them
 }
+
+rule arxan : obfuscator
+{
+  meta:
+    description = "Arxan"
+    info = "https://www.arxan.com/products/application-protection-mobile"
```

Comment:
I think it's a good to have the URL as part of the meta data, but I think the key should be URL.